src/pc/common/PaletteContainer.js

```diff
@@ -127,7 +127,7 @@ class IndirectPaletteContainer {
 class SingleValueContainer {
   constructor (options) {
     this.value = options?.value ?? 0
-    this.bitsPerValue = options?.bitsPerValue ?? constants.MIN_BITS_PER_BLOCK
```

Comment:
This is wrong, this value represents the `bitsPerValue` to use when _upgraded_ to a non-single value container. The default can be whatever, but you _must_ respect the option when passed.

Reply:
Fixed